runhouse/servers/cluster_servlet.py

```diff
@@ -303,23 +303,20 @@ async def aperiodic_cluster_checks(self):
                         api_server_url=api_server_url,
                         prev_end_log_line=prev_end_log_line,
                     )
-                    if logs_resp is None:
+                    if not logs_resp:
```

Comment:
i think this doesnt fully capture the difference betweeen having no logs to send and there being an error when sending the logs - would this work?

```
                    if logs_resp is None:
                        # No logs to send
                        break
                    elif logs_resp.status_code != 200:
                        logger.error(
                            f"{logs_resp.status_code}: Failed to send cluster logs to Den: {logs_resp.json()}"
                        )
```